docs/guides/use-env-with-platformatic.md

```diff
@@ -0,0 +1,84 @@
+# How to use ENV variables with Platformatic
+
+## Introduction
+
+Environment variables (env) are key-value pairs that can store data outside of your application code. This can include details such as API keys, and configuration options, which are isolated from your source code. 
+
+This guide provides a step-by-step guide on adding and using environment variables (env) in a Platformatic application.  
+
+## Setting up your Platformatic application
+
+First, if you haven't already, you need to set up your Platformatic application. [Run the command below](https://docs.platformatic.dev/docs/getting-started/quick-start-guide/), follow the steps, and initialize your application. 
+
+```bash
+npm create platformatic@latest
+```
+
+This will create a Platformatic project directory with all the files including `platformatic.json`, where you will store your `env` values.
+
+The `platformatic.json` file acts as the central configuration for your Platformatic application. Here, you define your database connection, server configurations, and how plugins should be loaded and configured with dynamic values from your environment variables. With this, your configurations can be easily changed without modifying your source code directly.
+
+## Adding options to your Plugin 
+
+In your Platformatic project, navigate to the [Plugins](https://fastify.dev/docs/latest/Reference/Plugins/) directory, and open the `example.js` file in your code editor. This file will generally be your starting point for new plugins.
+
+Next, add [options](https://fastify.dev/docs/latest/Reference/Plugins/#plugin-options) to your Plugins. Below is an example options code for a plugin:
+
+```json
```

Comment:
this is js, not json

Reply:
Apologies for missing this and thanks for pointing it out.